deployment/ansible/development/roles/docker_compose/templates/docker-compose.override.yml.j2

```diff
@@ -39,7 +52,7 @@ services:
       - VIRTUAL_HOST=users.inasafe.org
       - VIRTUAL_PORT=8080
       - BROKER_URL=amqp://guest:guest@rabbitmq:5672/
-      - INASAFE_REALTIME_BROKER_URL=amqp://guest:guest@{{ interpreters.inasafe_realtime.ipaddress }}:{{ docker_port_forward.inasafe_realtime.rabbitmq }}/
+      - INASAFE_REALTIME_BROKER_URL=amqp://guest:guest@{{ inasafe_realtime.broker }}:{{ inasafe_realtime.broker_port }}/
```

Comment:
@ismailsunni I moved some configuration to `inasafe_realtime` key because it might not be using pycharm interpreters now.

Reply:
Noted.